egs/wsj/s5/local/rnnlm/tuning/run_lstm_tdnn_1c.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)  Tony Robinson
+#           2017  Hainan Xu
+#           2017  Ke Li
+
+# This is the same as 1b, with two exceptions: TDNN is used as the acoustic model,
+# and the baseline is big-dict 4-gram LM.
+
+# System                    lstm_tdnn_1b lstm_tdnn_1c
```

Comment:
Please clarify what command produced this output.

Reply:
I'm not sure if we can specify an affix for the RNNLM type used, so both results are obtained using `local/chain/compare_wer.sh exp/chain/tdnn1h_sp` but in different `chain` dirs (named manually).

Reply:
I don't understand what you mean.  I think you should talk to Hainan or one of the other more senior students and come up with a more specific proposal (or just solve the problem some other way).  In any case, any time WERs are displayed, the command that generated those WERs should be displayed, or if it was stitched together manually somehow, then explain how.